fs/ext4/inode.c

```diff
@@ -3825,13 +3827,15 @@ static int __ext4_block_zero_page_range(handle_t *handle,
 
 	if (!buffer_uptodate(bh)) {
 		err = -EIO;
+		decrypt = S_ISREG(inode->i_mode) &&
+			ext4_encrypted_inode(inode);
+		ll_rw_block(REQ_OP_READ, (decrypt ? REQ_NOENCRYPT : 0), 1, &bh);
```

Comment:
This is still wrong compared to the original patch: https://android-review.googlesource.com/c/kernel/common/+/593373

There should only be one `ll_rw_block` call.

Reply:
Yupp, this is a mistake on my side. I'll force-push a fix now.

Reply:
Fixed it @nathanchance.